docs/content/tutorials/nginx-ingress-osm.md

```diff
@@ -0,0 +1,456 @@
+```
+
+### Verify that the envoy sidecar has been *injected* into NGINX Ingress
+
+```bash
+kubectl get pods -n nginx-ingress
+NAME                             READY   STATUS    RESTARTS       AGE
+nginx-ingress-7b9557ddc6-zw7l5   2/2     Running   1 (5m8s ago)   5m19s
+```
+
+2/2 shows we have two containers in the NGINX Ingress controller pod: NGINX Ingress and Envoy
+
+
+Configure your NGINX VirtualServer yaml to similar below
+
+```yaml
+apiVersion: k8s.nginx.org/v1
+kind: VirtualServer
+metadata:
+  name: httpbin
+  namespace: httpbin
+spec:
+  host: httpbin.example.com
+  tls:
+    secret: secret01
+  upstreams:
+  - name: httpbin
+    service: httpbin
+    port: 14001
+    use-cluster-ip: true
+  routes:
+  - path: /
+    action:
+      proxy:
+        upstream: httpbin
+        requestHeaders:
+          set:
+          - name: Host
+            value: httpbin.httpbin.svc.cluster.local
+```
+
+Test your configuration:
+
+```bash
+ curl  http://httpbin.example.com/get -v
+*   Trying 172.19.0.2:80...
+* TCP_NODELAY set
+* Connected to httpbin.example.com (172.19.0.2) port 80 (#0)
+> GET /get HTTP/1.1
+> Host: httpbin.example.com
+> User-Agent: curl/7.68.0
+> Accept: */*
+>
+* Mark bundle as not supporting multiuse
+< HTTP/1.1 200 OK
+< Server: nginx/1.23.3
+< Date: Sun, 19 Feb 2023 19:06:47 GMT
+< Content-Type: application/json
+< Content-Length: 454
+< Connection: keep-alive
+< access-control-allow-origin: *
+< access-control-allow-credentials: true
+< x-envoy-upstream-service-time: 2
+<
+{
+  "args": {},
+  "headers": {
+    "Accept": "*/*",
+    "Host": "httpbin.httpbin.svc.cluster.local",
+    "Osm-Stats-Kind": "Deployment",
+    "Osm-Stats-Name": "httpbin",
+    "Osm-Stats-Namespace": "httpbin",
+    "Osm-Stats-Pod": "httpbin-78555f5c4b-t6qln",
+    "User-Agent": "curl/7.68.0",
+    "X-Envoy-Internal": "true",
+    "X-Forwarded-Host": "httpbin.example.com"
+  },
+  "origin": "172.19.0.1",
+  "url": "http://httpbin.example.com/get"
+}
+* Connection #0 to host httpbin.example.com left intact
+```
+
+
+
+## Using The Open Service Mesh `ingressBackend` "proxy" Feature
+
+Install OSM into cluster.
+By running the following command, you will install OSM into the cluster with the mesh name `osm-nginx` using the `osm-system` namespace.
+
+```bash
+osm install --mesh-name osm-nginx --osm-namespace osm-system
+```
+
+Once OSM has been installed, this next command will mark the NGINX Ingress Controller as part of the OSM mesh, while also disabling sidecar injection.
+*NOTE*: The nginx-ingress name can be created as part of the NGINX Ingress install process, or manually. It must be created before you "add" the namespace to nginx-ingress.
+
+```bash
+osm namespace add nginx-ingress --mesh-name osm-nginx --disable-sidecar-injection
+```
+
+# You can now install NGINX Ingress Controller by one of th above methods listed: `helm` or `manifetsts`.
```

Comment:
th**e**

Also, why is this instruction below what we have already linked them off to?
If they need to install we should be telling them that just before we provide the links.

Reply:
Can remove them. It is a duplicate basically.